packages/cli/src/WorkflowExecuteAdditionalData.ts

```diff
@@ -902,10 +903,11 @@ async function executeWorkflow(
 		}
 		data = await workflowExecute.processRunExecutionData(workflow);
 	} catch (error) {
+		const executionError = error ? (error as ExecutionError) : undefined;
```

Comment:
What's the `error` we're throwing here? Is it possible to prevent it being thrown at all?

Reply:
That error has nothing to do with the bug reported, I just noticed that it was untyped. I guess anything that could go wrong during the execution would throw that particular error.